app/shared/design-system/components/primitives/createContentController.js

```diff
@@ -0,0 +1,106 @@
+/*
+	@example
+
```

Comment:
lets format comment with 2 spaces too

Reply:
would be also great to add a few sentences what this this is designed to do, might be not trivial to understand, same for others